.github/workflows/kubespider-style.yaml

```diff
@@ -42,10 +42,13 @@ jobs:
         ROOT=$(pwd)
 
         pylint $(git ls-files '*.py' | grep -v "downloaders/") --max-line-length=240
-        
+        skip_downloaders=(tiktok-dlp)
```

Comment:
I think this should be configured in `.pylintrc`, https://stackoverflow.com/questions/2503717/ignore-by-directory-using-pylint

Reply:
According to the CI script `.github/workflows/kubespider-style.yaml` lines 46-52, it is not possible to exclude using `.pylintrc` because all downloader files have the same structure, and we cannot differentiate specific downloader files.